pkg/cli/debug/debug.go

```diff
@@ -214,6 +214,7 @@ func NewCmdDebug(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra
 	cmd.Flags().BoolVarP(&o.DisableTTY, "no-tty", "T", o.DisableTTY, "Disable pseudo-terminal allocation")
 	cmd.Flags().StringVarP(&o.Attach.ContainerName, "container", "c", o.Attach.ContainerName, "Container name; defaults to first container")
 	cmd.Flags().BoolVar(&o.KeepAnnotations, "keep-annotations", o.KeepAnnotations, "If true, keep the original pod annotations")
+	cmd.Flags().BoolVar(&o.KeepLabels, "keep-labels", o.KeepLabels, "If true, keep the original pod labels")
```

Comment:
You've exposed this flag, but we don't copy those labels anywhere in the code, so that's why it's failing for @sallyom

Reply:
We do copy them:

https://github.com/openshift/oc/blob/c923633547ce429cab2903f063b4e8d423227894/pkg/cli/debug/debug.go#L795